tests/tests/test_utils.py

```diff
@@ -96,16 +99,56 @@ def test_serialize_exceptions(self) -> None:
             SyntaxError("Wrong"),
             ImproperlyConfigured("It's wrong"),
             InvalidTaskError(""),
-            SystemExit(),
         ]:
             with self.subTest(exc):
                 data = utils.exception_to_dict(exc)
                 self.assertEqual(utils.json_normalize(data), data)
-                self.assertEqual(set(data.keys()), {"exc_type", "exc_args"})
+                self.assertEqual(
+                    set(data.keys()), {"exc_type", "exc_args", "exc_traceback"}
+                )
                 reconstructed = utils.exception_from_dict(data)
                 self.assertIsInstance(reconstructed, type(exc))
                 self.assertEqual(reconstructed.args, exc.args)
 
+                # Check that the exception traceback contains only one line,
+                # the one with the name of the exception and the message.
+                dotted_path = utils.get_module_path(exc.__class__).replace(
+                    "builtins.", ""
+                )
+                msg = f": {exc.args[0]}\n" if exc.args[0] else "\n"
+                self.assertEqual(data["exc_traceback"], [dotted_path + msg])
+
+    def test_serialize_full_traceback(self) -> None:
+        try:
+            # Using optparse to generate an error because:
+            # - it's pure python
+            # - it's easy to trip down
+            # - it's unlikely to change ever
+            # - We can normalize the strack trace from it
+            optparse.OptionParser(option_list=[1])  # type: ignore
```

Comment:
Suggestion: Why not just `1 / 0`?

Reply:
It would tie the test to the code base, if the code base changes, the stack would change.

Since optparse is deprecated, but will likely stay in the stdlib for the next decade, it provides a stable stack trace to compare against for the test.

Reply:
I'd still expect the stacktrace to include the codebase, although I agree a deeper stack does help the test case a little more.

I think it'd be nice to have a flatter stack test too, perhaps also one which originates from C